data/json/recipes/other/parts.json

```diff
@@ -384,7 +384,7 @@
       { "proficiency": "prof_welding_basic", "fail_multiplier": 1.5 },
       { "proficiency": "prof_welding" }
     ],
-    "components": [ [ [ "material_aluminium_ingot", 8 ] ] ]
+    "components": [ [ [ "material_aluminium_ingot", 8 ], [ "scrap_aluminum", 35 ] ] ]
```

Comment:
Double checking that 8 ingots is convertible to 35 scrap?   How many scrap to make an ingot?